modules/Microsoft.Web/sites/.test/webAppCommon/deploy.test.bicep

```diff
@@ -153,5 +153,9 @@ module testDeployment '../../deploy.bicep' = {
     userAssignedIdentities: {
       '${nestedDependencies.outputs.managedIdentityResourceId}': {}
     }
+    sitePublishingCredPolicyNames: [
```

Comment:
Update accordingly as an array of objects

Reply:
Updated